### PR TITLE
Connect Redux To React Components 

### PR DESCRIPTION
**Pull request by Chimoneg27**
### For this feature I did the following

- Made sure to remove linter errors
- Connected my redux store to my react code with a provider
- Used the redux hooks to connect with my React components
- Created a separate button component that submits the book

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+  const book = useSelector((state) => state.books.books);
+  const dispatch = useDispatch();
+
+  return (
+    <div className="home-page">
+      <Navbar />
+      <h1>Welcome Home</h1>
+      <div className="bookshelf">
+        {book.map((book) => (
+          // eslint-disable-next-line camelcase
```
Comment (prolajumokeoni) [inline src/components/Home.js]: - [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks, please remove it 👍🏿

File: src/components/Button.js [hunk 2]
```diff
@@ -2,9 +2,9 @@
 import React from 'react';
 import PropTypes from 'prop-types';
```
Comment (prolajumokeoni) [inline src/components/Button.js:3]: - [ ] Kindly avoid disabling linters as it is not a good practice in line 1, it is better for your linters to pass the linter checks, please remove it 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Chimoneg27 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Implemented correctly the functions to remove and add book  ✅ 
-  Defined an initial state slice that contains books and added to booksSlice 🚀 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Chimoneg27,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Implement the requirements of the previous reviewer
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**